tcms/testruns/views.py

```diff
@@ -861,8 +851,6 @@ def get_context_data(self, **kwargs):
             'test_case_run_bugs': bug_ids,
             'mode_stats': mode_stats,
             'summary_stats': summary_stats,
-            'jira_url': jira_url,
-            'bugzilla_url': bugzilla_url
```

Comment:
These context variables are used in the template to render a single URL which will open all the bugs at once. The previous hard-coded design distinguished between Bugzilla and JIRA only by the presence of the dash symbol ("-") inside the bug ID. This is a terrible design and it doesn't really work with the current changes.

When viewing the test run bug report you can still see bugs for individual test case runs, you also see a list of all unique bugs at the bottom of the screen. However you no longer see the 2 shortcut URLs which will pen the list of all bugs inside a single page in the bug tracker.

This is similar to my previous comment, I will create a new issue to track its progress but will not add any more changes related to this to the current PR because it is already big enough.